src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -4540,7 +4540,7 @@ interface IOptional
 
             class Opt : IOptional
             {
-                int IOptional.Goo(int g)
+                int IOptional.Goo(int g = 0)
```

Comment:
what happened here?  explicit interface members should not have default values.

Reply:
(compiler will warn if you do this).

Reply:
SharpLab shows that only a warning is shown indicating that the default value will have no effect. Though it's better to never generate the default value. I'll change this.

Reply:
This has been reverted.